src/main/java/org/libsdl/app/SDLActivity.kt

```diff
@@ -247,7 +247,7 @@ open class SDLActivity internal constructor (context: Context?) : RelativeLayout
         mSurface.setOnTouchListener(this)
     }
 
-    override fun surfaceCreated(holder: SurfaceHolder?) {
```

Comment:
why did this change actually, is this because of an SDK update? we should probably change the samples app as well then @rikner

Reply:
Yeah, pretty sure it was because of the SDK update 👍